### PR TITLE
ER-492: Added DDB-widget module

### PR DESCRIPTION
**Pull request by cableman**
Screenshot 2019-05-14 at 11 52 36

### CODE REVIEW

File: sites/all/modules/ddb_widget/ddb_widget.module [hunk 1]
```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * @file
+ * Extend the site with information about titles hosted at overdrive.
```
Comment (rimi-itk) [inline sites/all/modules/ddb_widget/ddb_widget.module]: That's surprising!

File: sites/all/modules/ddb_widget/includes/ddd_widget.admin.inc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    '#type' => 'checkboxes',
+    '#title' => t('Show block for specific roles'),
+    '#default_value' => $config['role']['roles'],
+    '#options' => $role_options,
+    '#description' => t('Show this block only for the selected role(s). If you select no roles, the block will be visible to all users.'),
+  ];
+
+
+  return system_settings_form($form);
+}
```
Comment (rimi-itk) [inline sites/all/modules/ddb_widget/includes/ddd_widget.admin.inc]: Please add new line at end of file.

File: sites/all/modules/ddb_widget/ddb_widget.module (hunk 1 above)
Comment (cableman) [reply sites/all/modules/ddb_widget/ddb_widget.module]: Fixed

File: sites/all/modules/ddb_widget/includes/ddd_widget.admin.inc (hunk 2 above)
Comment (cableman) [reply sites/all/modules/ddb_widget/includes/ddd_widget.admin.inc]: Fixed

**Pull request merged**